ps-pxc-dist/rpm_release.sh

```diff
@@ -0,0 +1,67 @@
+set -x
+set -e
+#
+DIRECTORIES=""
+
+# Create list of packages to push:
+for dir in $(printenv | grep _PATH | sed 's/^.*=//g'); do
+    DIRECTORIES+=" $dir"
+done
+
+if [ -z $DIRECTORIES ]; then
+    echo "No PATH specified..."
+    exit 1
+fi
+#
+REPOCOMP=$(echo "${COMPONENT}" | tr '[:upper:]' '[:lower:]')
+REPOPATH=${REPOSITORY}
+#
+if [[ ${REPOSITORY} == "PDPXC" ]]; then
+  REPOPATH_TMP=repo-copy/pdpxc-${REPOSITORY_VERSION}/yum
+  if [ $REPOSITORY_VERSION_MAJOR == true ]; then
+    REPOPATH_TMP+=" repo-copy/pdpxc-$(echo ${REPOSITORY_VERSION} | awk -F"." 'BEGIN { OFS = "." }{ print $1, $2}')/yum"
+  fi
+fi
+#
+if [[ ${REPOSITORY} == "PDPS" ]]; then
+  REPOPATH_TMP=repo-copy/pdps-${REPOSITORY_VERSION}/yum
+  if [ $REPOSITORY_VERSION_MAJOR == true ]; then
+    REPOPATH_TMP+=" repo-copy/pdps-$(echo ${REPOSITORY_VERSION} | awk -F"." 'BEGIN { OFS = "." }{ print $1, $2}')/yum"
+  fi
+fi
+#
+for REPOPATH in $REPOPATH_TMP; do
+    for $dir in $DIRECTORIES; do
+        #
+        cd /srv/UPLOAD/$dir
+        #
+        # getting the list of RH systems
+        RHVERS=$(ls -1 binary/redhat)
+        #
+        # source processing
+        SRCRPM=$(find source/redhat -name '*.src.rpm')
```

Comment:
please add check if source directory is available.
There are some cases once we don't have it. So we should push packages without source packages